url-plugins/widgets/UrlFetch-realtimesource.json

```diff
@@ -0,0 +1,76 @@
+{
+  "metadata": {
+    "spec-version": "1.0"
+  },
+  "configuration-groups": [
+    {
+      "label": "URL Fetch Configuration",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "URL",
+          "name": "url"
+        },
+        {
+          "widget-type": "number",
+          "label": "Interval",
+          "name": "interval",
+          "widget-attributes" : {
+            "default": "3600",
+            "min": "1"
+          }
+        },
+        {
+          "widget-type": "keyvalue",
```

Comment:
if we change this to be json instead of key-value, the widget-type will be json-editor.